assets/blocks/course-outline/store.js

```diff
@@ -38,13 +38,19 @@ const actions = {
 				data: { structure: yield getEditorStructure() },
 			} );
 			yield actions.setStructure( result );
+
+			yield dispatch( 'core/notices' ).removeNotice(
+				'course-outline-save-error'
+			);
 		} catch ( error ) {
-			yield dispatch( 'core/notices' ).createErrorNotice(
-				[
-					__( 'Course Outline update failed:', 'sensei-lms' ),
-					error.message,
-				].join( ' ' )
+			const errorMessage = sprintf(
+				/* translators: Error message. */
+				__( 'The course could not be saved. %s', 'sensei-lms' ),
```

Comment:
I used the word `saved` instead of `updated` (suggested in the issue) because an error can happen also on publishing for the first time.